efel/cppcore/LibV1.cpp

```diff
@@ -271,7 +272,7 @@ int LibV1::firing_rate(mapStr2intVec& IntFeatureData,
     return nSize;
   else {
     vector<double> stimStart, stimEnd, peakVTime, firing_rate;
-    double lastAPTime;
+    double lastAPTime = 0.;
```

Comment:
how do you know 0 is a good value for no spikes?
shouldn't  there be a special condition at the end "if(nCount == 0)" ?